orangecontrib/spectroscopy/data.py

```diff
@@ -17,6 +17,8 @@
 from .pymca5 import OmnicMap
 from .agilent import agilentImage, agilentMosaic, agilentImageIFG, agilentMosaicIFG
 
+SET_flow = False    # Use for manual control flow in some function inter Class
+                    # See example in class NeaSpec and GSFReader
```

Comment:
@markotoplak  I created this global variable to use GSF reader, because I needed a return before the default return.
I think this approach can be used in many other cases.
But if you have any suggestion, teel me

Reply:
I answered with a suggestion about making a new function. For now, please avoid using global.